CHANGELOG.rst

```diff
@@ -6,7 +6,13 @@ This document describes changes between each past release.
 3.3.0 (unreleased)
 ==================
 
-- Nothing changed yet.
+**Protocol**
+
+- Add new *experimental* endpoint ``GET /v1/permissions`` to retrieve the list of permissions
+  granted on every kind of object (#600).
+  Requires setting ``kinto.experimental_permissions_endpoint`` to be set to ``true``.
```

Comment:
I like the fact that it warns the user that its enabling an experimental feature. Let's remove that when kinto-admin pull-request is ready